src/crsAdapter/CetsAdapter.js

```diff
@@ -551,6 +605,49 @@ class CetsAdapter {
         });
     }
 
+    assignHotelServiceFromAdapterObjectToXmlObject(service, xml) {
+        let startDate = moment(service.startDate, this.options.useDateFormat);
+
+        let xmlService = {
+            [CONFIG.builderOptions.attrkey]: {
+                ServiceType: CONFIG.defaults.serviceType.hotel,
+            },
+            Product: service.destination.substring(3),
+            Program: 'HOTEL',
+            Destination: service.destination.substring(0, 3),
+            Room: service.roomCode,
+            Norm: service.roomOccupancy,
+            MaxAdults: service.roomQuantity,
+            Meal: service.mealCode,
+            StartDate: startDate.isValid() ? startDate.format(CONFIG.crs.dateFormat) : service.dateFrom,
+            Duration: this.calculateDuration(service.dateFrom, service.dateTo),
+        };
+
+        xml.Fah.push(xmlService);
+    }
+
+    assignHotelTravellers(service, xml) {
+        if (!service.travellers) return;
+
+        xml.Fap = [];
+
+        service.travellers.forEach((serviceTraveller, index) => {
+            const traveller = this.helper.traveller.normalizeTraveller(serviceTraveller);
+            
+            const travellerName = traveller.name.split(' ');
+            xml.Fap.push({
+                [CONFIG.builderOptions.attrkey]: {
+                    ID: index + 1,
+                },
+                PersonType: traveller.salutation,
+                Name: travellerName[0] + ' ' + travellerName[1],
```

Comment:
i guess this should only be the last name: `travellerName[1]`

Reply:
Absolutely right!